### DISCUSSION TITLE
Add tools to download 'tb calc' from PHIUS website endpoints etc?

### DISCUSSION THREAD

**Discussion by TfedUD** [PH-Tools/honeybee_ph#53] [Ideas]
Hi! I wish we could like 'git clone directory with all the crap setup that even has a readme with instructions and excerpts from ISO standards' stuff. 

But anyways: I want to add some tooling that uses select box to choose what calc but "input folder path, select calc, press button to download"
stuff like this endpoint:
https://www.phius.org/sites/default/files/2022-05/Thermal_Bridge_Psi-Value_Calculator_v2.4.xlsx

Maybe in phx initially so you can like CLI yourself a PHIUS project directory?
that would be HELLA cool imo...

but then use said CLI; hook it up to a Gh component too

**Comment by ed-p-may** [comment]
Yeah creating and exported to the Excel tool would be great. Could use the same pattern as PHX's 'to_phpp' using XLwings as the engine? Sadly the input data can't be extracted from Flixo automagically (they lock that down unless you buy their suepr-pro edition). But this seems like its made for THERM anyway, not Flixo, right? So could try and use the THERM tools for reporting data and grab / collect the data needed and then dump into the spreadhsheet. I think that would be very useful for Phius if they are going to insist on using this reporting tool.

**Reply by TfedUD** [reply]
yeah! that's what I was thinking: I was going to borrow from your stuff for doing the faces : assemblies visuals and stuff:
   The 'how to' do the whole nine yards in terms of getting the images and numbers etc into the xlsx!
   
  Speaking of xlwings... I wish PHIUS saved all their calcs as the flat file version of excel on like git for src control and we could just 'xw.makebookfromthisfilething()' off git+current_release_calc.xlsx
